pkg/runtime/namespace_config_provider.go

```diff
@@ -0,0 +1,40 @@
+package runtime
```

Comment:
does this file need to be updated?

Reply:
This is already updated and basically doing the mapping. So if the value is "domain" in the config it will return a Domain enum value.

Reply:
oh I mean when I pull this locally this file doesn't compile